sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -271,6 +297,9 @@ export class EventHubReceiver extends LinkEntity {
     };
 
     this._onSessionError = (context: EventContext) => {
+      if (this._aborter) {
+        this._aborter.removeEventListener("abort", this._onAbort);
+      }
```

Comment:
`_onSessionClose` will be called right after `_onSessionError` (that's why we have the detached() being called in _onSessionClose)

So, we don't need to remove the listener here as you are already doing that in _onSessionClose